src/cmds/scripts/pbs_habitat.in

```diff
@@ -82,11 +82,6 @@ chk_dataservice_user() {
 }
 
 check_switch() {
-	adapters=`ntblstatus 2> /dev/null | awk '/^Adapter/ {print $2}' | uniq`
```

Comment:
Please remove `check_switch` function itself and no one uses it in this file, and if I remember then this function is moved to pbs_postinstall.